test/modules/storage-aws-s3/snapshot_create_test.go

```diff
@@ -40,7 +40,9 @@ func Test_S3Storage_StoreSnapshot(t *testing.T) {
 
 		endpoint := os.Getenv(minioEndpoint)
 		s3Config := s3.NewConfig(endpoint, "bucket", false)
-		s3, err := s3.New(s3Config, logger)
+		s3, err := s3.New(s3Config, logger, "dummyString")
+		path, _ := os.Getwd()
+		s3.SetDataPath(path)
```

Comment:
you can pass the `dataPath` directly to `s3.New(...)` no need for `s3.SetDataPath` method